t/writer.t

```diff
@@ -72,6 +72,23 @@ subtest 'oddballs and regressions' => sub{
       like $@, qr/found undefined value/;
 
   };
+
+  subtest 'quoted inline table keys' => sub {
+
+      my $data = { q{foo} => [ q{bar}, { q{<=} => 33 } ] } ;
+      my $encoded = to_toml( $data );
+
+      my $decoded;
+      ok( lives { $decoded = from_toml( $encoded ) },
+          'decode succeeded' ) or note $@;
+
+      SKIP: {
+            skip 'error in encoding' unless $decoded;
+            is ( $decoded, $data, 'round trip successful' )
+        }
```

Comment:
Why does this need a SKIP block?

Reply:
If the decode failed (caught by the previous test) this test will always fail; in that case running this test creates output indicating that there are two failing tests, when there really is one.  (Kind of like a single syntax error in code causes an avalanche of useless errors that just confuse things.)

Reply:
If your question is "why use a SKIP block instead of a simple conditional" I like the fact that the output indicates that a test was skipped because of a prior failing test.

Reply:
I asked in #toolchain on irc.perl.org and it was confirmed that this is an unusual way of using a `SKIP` block. I would normally expect to see a `SKIP` if a test is for a specific environment or if some library is missing. I think I haven't seen it used to avoid cascading errors before. If the user cares about this, they could always enable some kind of "bail on fail" behaviour when running the tests.

So, I think it's clearer without the `SKIP` block.

Reply:
Ok.  (I've had to deal with really poorly written test suites where tons of errors were caused by a single bad test; debugging that was a PITA, as figuring out whether the errors were related was not straightforward. )